orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -38,17 +39,74 @@ def _ortvalue_from_torch_tensor(torch_tensor):
     return C.OrtValue.from_dlpack(to_dlpack(torch_tensor), is_bool_tensor)
 
 
-def _torch_tensor_from_dl_pack(dlpack, ortvalue, device):
-    torch_tensor = from_dlpack(dlpack) if device.type != 'ort' else C.ort_from_dlpack(dlpack)
-    return torch_tensor.to(torch.bool) if ortvalue.data_type() == 'tensor(bool)' else torch_tensor
+def _ortvalues_to_torch_tensor(ortvalues, device):
```

Comment:
This function need to return a pytorch tensor. 
In the original code,  "C.ort_from_dlpack(dlpack)" actually returns a pytorch tensor. (I know the naming here is confusing, but it's returning a at::Tensor).
In the modified code, "C.OrtValue.from_dlpack()" is actually returning a ORTValue.